solr/core/src/java/org/apache/solr/core/ConfigOverlay.java

```diff
@@ -34,15 +34,15 @@
  * performed on tbhis gives a new copy of the object with the changed value
  */
 public class ConfigOverlay implements MapSerializable {
-  private final int znodeVersion;
+  private final int version;
```

Comment:
Yep, because the version might not be a znode version, it might be an ending to the lastModified date from the filesystem (using a filesystem configset). There are other znodeVersion names I could have changed, but that would have been a lot more far-ranging than just changing this one for now.

In general for things that don't have to be a ZKConfigSet we should move from znodeVersion to version where feasible.